api/route/base.go

```diff
@@ -44,16 +46,18 @@ func SetUpRouter() *gin.Engine {
 	AppendHealthCheck(r)
 	AppendAuthentication(r)
 	//AppendRoute(r)
-	AppendSsl(r)
+	//AppendSsl(r)
 	AppendPlugin(r)
-	AppendUpstream(r)
+	//AppendUpstream(r)
```

Comment:
Why not remove them directly?

Reply:
we need them to debug or compare.will remove them once when complete the refactoring work.